### PR TITLE
chore(ci) fix large CI workflow file

### PR DESCRIPTION
**Pull request by casimiro**
[This](https://github.com/Kong/ngx_wasm_module/actions/runs/7160665577) successful run demonstrates the fix.

### CODE REVIEW

**Comment by thibaultcha** [comment]
Glad that you found the root cause, awesome. How could we fix it without removing the ability to run `job-build-tests.yml` with different `os`? E.g. `macos-latest` in the near future?

**Comment by casimiro** [comment]
While addressing the issue I concluded that the problem was the single-entry bracket-list and that having more than one entry, e.g. `os: [ubuntu-latest, macos-latest]`, wouldn't trigger the error.

But I actually haven't tested adding more entries to that list to confirm. I'll do that.

If having more items on that list also solves the issue, we can simply put the variable `os` back to the matrix containing `ubuntu-latest` and the added OS, e.g. `os: [ubuntu-latest, macos-latest]`, when we want to add a new OS.

File: .github/workflows/ci-large.yml [hunk 1]
```diff
@@ -139,7 +139,7 @@ jobs:
         v8: [11.4.183.23]
         include:
           - label: old_nginx
-            os: [ubuntu-latest]
+            os: [ubuntu-latest, macos-latest]
```
Comment (thibaultcha) [inline .github/workflows/ci-large.yml:142]: Hmm iirc we can't have array values in `include`?

Comment (casimiro) [reply .github/workflows/ci-large.yml:142]: Hm, I think you are right. I'll update the commit accordingly.

**Comment by casimiro** [comment]
The fix can be verified in [this](https://github.com/Kong/ngx_wasm_module/actions/runs/7171060391) successful run.

**Pull request merged**